.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,7 +11,7 @@ no particular length requirements.
 
 If you need to write multiple paragraphs, feel free.
 
-LP: #NNNNNNN (replace with the appropriate bug reference or remove
```

Comment:
I'd prefer we still document that if the issue is tagged launhpad and has an LP reference, we still want an `LP: #ID` footer supplement as well as the `Fixes: #NNNN`. Then launchpad automation will still mark the "expired" bug as fix released for us once we publish to Ubuntu. This gives launchpad bug filers a feedback mechanism that state has changed toward resolution for the bug.

Reply:
+LP: #NNNN (if a source Launchpad bug is associated with the GitHub issue fixed)

Reply:
I don't think Launchpad automation does that for upstream bugs. I think that's only for `Ubuntu(cloud-init)`. That's why [this](https://github.com/canonical/uss-tableflip/blob/main/doc/upstream_release_process.md#close-bugs) has been part of our upstream release process.

Reply:
Ohh sorry, right, but that part of the release process relies on us processing the git log commit messages  to obtain said LP: # numbers. Do we not want to represent the released context to LP in this case? It seems to be a good way to close the loop on an otherwise 'Expired' bug status.